src/lib/UploadcareFile.ts

```diff
@@ -68,4 +69,8 @@ export class UploadcareFile {
         mime_type: this.mimeType,
         size: this.size,
     });
+
+    private static replaceForbiddenSymbols = (filename: string) => {
```

Comment:
Why do you need this method? 🤔 Can't you simply inline the `sanitizeFilename` line? Or just have it as a util function?